packages/registry/gulpTasks/copySchemasTask.js

```diff
@@ -0,0 +1,34 @@
+const gulp = require('gulp');
+const path = require('path');
+const jsonCombine = require('gulp-jsoncombine');
+
+const taskPaths = require('./taskPaths');
+
+const mainSchemaFilename = 'profile.schema.json';
+
+function combineSchemas(input) {
+  const mainSchemaKey = path.basename(mainSchemaFilename, '.json');
+  const mainSchema = input[mainSchemaKey];
+  const dependencies = [];
+  Object.keys(input).forEach((key) => {
+    if (key !== mainSchemaKey) {
+      dependencies.push(input[key]);
+    }
+  });
+
+  const schemas = {
+    mainSchema,
+    dependencies
+  };
+  return Buffer.from(JSON.stringify(schemas, null, 2));
```

Comment:
Silly nit: I had to go searching for what the magic number `2` meant here. Any chance you could use something like `const indentationSpaces = 2` here and in the other `JSON.stringify()` calls to assist with readability?

Reply:
I currently use this in 28 of places throughout the repo.  Is this a change you're hoping for across the board?